sky/serve/service_spec.py

```diff
@@ -77,6 +79,13 @@ def __init__(
                     'Currently, SkyServe will cleanup failed replicas'
                     'and auto restart it to keep the service running.')
 
+        # Add the check for unknown load balancing policies
+        if load_balancing_policy is not None and \
+            load_balancing_policy not in LB_POLICIES:
```

Comment:
```suggestion
        if (load_balancing_policy is not None and
            load_balancing_policy not in LB_POLICIES):
```
nit: Do not use a backslash for explicit line continuation. Instead, make use of Python’s implicit line joining inside parentheses, brackets and braces.
https://google.github.io/styleguide/pyguide.html#32-line-length